index.d.ts

```diff
@@ -564,4 +594,18 @@ declare module 'warframe-items' {
         'Warframe' |
         'Whip' |
         '---'
+        
+    type Element = 
+        'Impact' |
+        'Puncture' |
+        'Slash' |
+        'Toxin' |
+        'Electricity' |
+        'Blast' |
+        'Radiation' |
```

Comment:
What do you think of already adding Gas so we don't have to update the types when a weapon of that type does launch?

Reply:
Also we can use the DamageTypes interface and just use Element as an alias:
`type Element = Capitalize<keyof DamageTypes>;`

Reply:
Feel free to use the code suggestions so I can merge them from here.

Reply:
Like I've mentioned, I don't use the types, so this is entirely up for your convenience